ui/src/main/webapp/wizards/ldap/stages/bind-settings.js

```diff
@@ -58,39 +77,56 @@ const BindSettings = (props) => {
         The requirements for different LDAP servers vary; please contact your LDAP administrator if you need guidance.
       </Description>
 
-      <Input id='bindUser' disabled={disabled} label='Bind User' />
-      <Password id='bindUserPassword' disabled={disabled} label='Bind User Password' />
-      <Select id='bindUserMethod'
-        label='Bind User Method'
-        disabled={disabled}
-        options={bindUserMethodOptions} />
-      {/* removed options: 'SASL', 'GSSAPI SASL' */}
-      {/* TODO GSSAPI SASL only */}
-      {/* <Input id='bindKdcAddress' disabled={disabled} label='KDC Address (for Kerberos authentication)' /> */}
-      {/* TODO GSSAPI and Digest MD5 SASL only */}
-      {/* Realm is needed for Kerberos and MD5 auth, currently only MD5 is supported by the wizard */}
-      <Input visible={bindUserMethod === 'Digest MD5 SASL'} id='bindRealm'
-        disabled={disabled} label='Realm (for Digest MD5 authentication)' />
-
-      <ActionGroup>
-        <Action
-          secondary
-          label='back'
-          onClick={prev}
-          disabled={disabled} />
-        <Action
-          primary
-          label='next'
-          onClick={test}
+      <Body>
+        <Input id='bindUser' disabled={disabled} label='Bind User' />
+        <Password id='bindUserPassword' disabled={disabled} label='Bind User Password' />
+        <Select id='bindUserMethod'
+          label='Bind User Method'
           disabled={disabled}
-          nextStageId='directory-settings'
-          testId='bind' />
-      </ActionGroup>
+          options={bindUserMethodOptions} />
+        {/* removed options: 'SASL', 'GSSAPI SASL' */}
+        {/* TODO GSSAPI SASL only */}
+        {/* <Input id='bindKdcAddress' disabled={disabled} label='KDC Address (for Kerberos authentication)' /> */}
+        {/* TODO GSSAPI and Digest MD5 SASL only */}
+        {/* Realm is needed for Kerberos and MD5 auth, currently only MD5 is supported by the wizard */}
+        <Input visible={bindUserMethod === 'Digest MD5 SASL'} id='bindRealm'
```

Comment:
❗️ This string doesn't match the expected value to map to the enum on the backend - `DigestMD5SASL` - as seen [here](https://github.com/connexta/admin-console-beta/blob/master/query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/connection/LdapBindMethod.java#L63).